charts/trickster/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 # Note:
 # - Starting at 2.0.0 so that chart release is not confused with chart built for v1.x release
 # - This version is the chart version and is different from the trickster application version
-version: 2.0.0
+version: 2.1.0
```

Comment:
can we keep this at 2.0.0 since the chart is not used/published as a release yet?